<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="hp" name="com.test.ExtentReporting.ExtentReportingDemowithTestNG" tests="1" failures="0" timestamp="25 Mar 2020 05:51:00 GMT" time="6.571" errors="1">
  <testcase name="demotest" time="6.571" classname="com.test.ExtentReporting.ExtentReportingDemowithTestNG">
    <error type="java.io.IOException" message="Media was not found at [./ExtentReport/result.png]">
      <![CDATA[java.io.IOException: Media was not found at [./ExtentReport/result.png]
at com.aventstack.extentreports.reporter.BasicFileReporter.mediaExists(BasicFileReporter.java:248)
at com.aventstack.extentreports.reporter.BasicFileReporter.onScreenCaptureAdded(BasicFileReporter.java:236)
at com.aventstack.extentreports.ExtentObservable.addScreenCapture(ExtentObservable.java:360)
at com.aventstack.extentreports.ExtentTest.addScreenCaptureFromPath(ExtentTest.java:1125)
at com.aventstack.extentreports.ExtentTest.addScreenCaptureFromPath(ExtentTest.java:1141)
at com.test.ExtentReporting.ExtentReportingDemowithTestNG.demotest(ExtentReportingDemowithTestNG.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- demotest -->
</testsuite> <!-- com.test.ExtentReporting.ExtentReportingDemowithTestNG -->
